DogeNetWebClient/src/app/friends/pages/friends-grid/friends-grid.component.html

```diff
@@ -0,0 +1,14 @@
+<div class="grid">
+  <app-friend-card></app-friend-card>
```

Comment:
Why you didn't mock friend list in component instead?

Reply:
done